config/csm/rest_response_data.yaml

```diff
@@ -62,6 +63,26 @@
     - "Invalid request body: {'role': ['Must be one of: admin, manage, monitor.']}"
     - "Invalid request body: {'role': ['Must be one of: admin, manage, monitor.'], 'password': [\"Password Policy Not Met. Must be more than 8 characters. Must contain at least one Uppercase Alphabet. Must contain at least one Lowercase Alphabet. Must include !@#$%^&*()_+-=[]{}|'.\"]}"
     - "Invalid request body: {'password': [\"Password Policy Not Met. Must be more than 8 characters. Must contain at least one Uppercase Alphabet. Must contain at least one Lowercase Alphabet. Must include !@#$%^&*()_+-=[]{}|'.\"], 'role': ['Must be one of: admin, manage, monitor.']}"
+    - "Invalid request body: {'password': ['Password Policy Not Met. Must contain at least one Numeric value.']}"
+    - "{'sortby': ['Must be one of: user_id, username, email, user_type, role, created_time, updated_time, email_address, user_role.']}"
+    - "{'dir': ['Must be one of: desc, asc.']}"
+    - "username cannot be modified"
+    - "Insufficient information in request body"
+    - "_schema: Empty value for access_key"
+    - "_schema: Empty value for secret_key"
+    - "Uid: Field may not be null."
+    - "Key_type: Must be one of: s3."
+    - "Key_type: Field may not be null."
+    - "max_buckets: not a valid integer."
+    - "max_buckets: field may not be null."
```

Comment:
please use templates for all these i.e. used in same file, line 57

Reply:
ok Sure Thanks @RajeshPratapSInghChouhan

Reply:
Updated the same cc @RajeshPratapSInghChouhan.